README.md

```diff
@@ -1,17 +1,17 @@
 # Steps for Titanium RTL Application:
-1 - Use this module to force language change, it'll load correct strings [link](http://shareourideas.com/2013/12/02/titanium-locale-module-for-both-android-and-ios/)
```

Comment:
Why you removed this, this one still force app to load specific language file and force native RTL in iOS @AhmedMSayed